src/RemixDevTools/RemixDevTools.tsx

```diff
@@ -42,22 +46,23 @@ const RemixDevTools = ({ defaultOpen, position }: Props) => {
           position === "top-right" && "rdt-right-0 rdt-top-0",
           position === "top-left" && "rdt-left-0 rdt-top-0",
           position === "middle-right" &&
-            "rdt-right-0 rdt-top-1/2 -rdt-translate-y-1/2",
+          "rdt-right-0 rdt-top-1/2 -rdt-translate-y-1/2",
           position === "middle-left" &&
-            "rdt-left-0 rdt-top-1/2 -rdt-translate-y-1/2"
+          "rdt-left-0 rdt-top-1/2 -rdt-translate-y-1/2",
+          isOpen && "rdt-hidden" // Hide the button when the dev tools is open
         )}
       >
         <Logo
           className={clsx(
             "rdt-h-14 rdt-w-14 rdt-rounded-full rdt-transition-all rdt-duration-200",
-            "rdt-hover:cursor-pointer rdt-hover:ring-2 rdt-ring-slate-600"
+            "rdt-hover:cursor-pointer rdt-hover:ring-2 rdt-ring-slate-600",
           )}
         />
       </div>
       <div
         style={{ zIndex: 9998, height }}
         className={clsx(
-          "rdt-duration-600 rdt-fixed rdt-bottom-0 rdt-left-0 rdt-box-border rdt-flex rdt-w-screen rdt-resize-y rdt-flex-col rdt-overflow-auto rdt-bg-[#212121] rdt-text-white rdt-opacity-0 rdt-transition-all",
+          "remix-dev-tools-window rdt-duration-600 rdt-fixed rdt-bottom-0 rdt-left-0 rdt-box-border rdt-flex rdt-w-screen rdt-resize-y rdt-flex-col rdt-overflow-auto rdt-bg-[#212121] rdt-text-white rdt-opacity-0 rdt-transition-all",
```

Comment:
You can remove this class and the css in the css file and just remove the rdt-resize-y class from here, I left it in by accident, that should remove the resize on the bottom right

Reply:
👍